examples/managed_instance_group/variables.tf

```diff
@@ -38,30 +35,37 @@ variable "mig_instance_count" {
 
 variable "image" {
   description = "The Docker image to deploy to GCE instances"
+  default     = "gcr.io/google-samples/hello-app:1.0"
 }
 
 variable "image_port" {
   description = "The port the image exposes for HTTP requests"
-}
-
-variable "restart_policy" {
-  description = "The desired Docker restart policy for the deployed image"
-}
-
-variable "machine_type" {
-  description = "The GCP machine type to deploy"
+  default     = 8080
 }
 
 variable "region" {
   description = "The GCP region to deploy instances into"
 }
 
-variable "zone" {
-  description = "The GCP zone to deploy instances into"
+variable "network" {
+  description = "The GCP network"
+  default     = "mig-net"
 }
 
 variable "additional_metadata" {
   type        = "map"
   description = "Additional metadata to attach to the instance"
   default     = {}
 }
+
+variable "service_account" {
+  type    = object({
+    email  = string,
+    scopes = list(string)
+  })
+  default = {
+    email  = ""
+    scopes = [
```

Comment:
Better set it in one-line `scopes = ["cloud-platform"]`

Reply:
Thank you for review, @paddycarver! [Fixed](https://github.com/terraform-google-modules/terraform-google-container-vm/compare/b6d0f962d7a2074d4392d31a3368b8c6808df77b..1c02f6b6764e3d373d58b2fb60bf197b17cf6f7a)